montblanc/examples/MS_tf_example.py

```diff
@@ -84,6 +85,9 @@ def updated_dimensions(self):
         """ Tell montblanc about dimension sizes (point sources only) """
         return [('npsrc', self._npsrc)]
 
+    def phase_centre(self, context):
+        return np.array([0.0, -10.0])
+
```

Comment:
This looks like this should be changed to something that is not hard-coded.

Reply:
@bennahugo I think it's important to address this particular issue though!

Reply:
If this is a test case, could the `phase_centre` more physically realistic? `-10` is quite high, maybe `1e-5`?

Reply:
this was debugging. yes it should be removed.